modules/gapi/src/backends/common/serialization.cpp

```diff
@@ -928,7 +928,7 @@ IIStream& ByteMemoryInStream::operator>> (std::string& str) {
     if (sz == 0u) {
         str.clear();
     } else {
-        str.resize(sz);
+        str.resize(std::size_t(sz));
```

Comment:
static_cast by code style(?)

Reply:
Yes, I think you're right. Just really don't like to write it.
Maybe there are no such strict conventions in OCV, but you're still right :)

Reply:
it is the place where other reviewers will pay attention, personally i dislike such writing long-symbol cast too, but it's still strong recommendation